src/Orleans/Runtime/GrainMethodInvocationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.CodeGeneration;
+using Orleans.Concurrency;
+
+namespace Orleans.Runtime.GrainMethodInvocationExtensions
+{
+    public static class GrainMethodInvocationExtensions
```

Comment:
Does anyone else want to weigh in here? My opinion is to let it be accessible, but it's not a firm opinion.